update_vendored_collections/update_vendored_collections.py

```diff
@@ -104,13 +104,24 @@ def spec_add_changelog(content, collection_tarballs, lines):
             return content
 
 
+def spec_bump_release(content):
+    print("Bumping release in the spec file")
+    cur_release = re.search(r"^Release: (\d+)", content, flags=re.MULTILINE).group(1)
+    new_release = str(int(cur_release) + 1)
+    content = re.sub(
+        r"^Release: \d+", "Release: " + new_release, content, flags=re.MULTILINE
+    )
+    return content
```

Comment:
You also need to update the release number in the changelog entry

Reply:
Script does this in spec_add_changelog: https://github.com/linux-system-roles/auto-maintenance/blob/master/update_vendored_collections/update_vendored_collections.py#L95